_visualisation/changelog.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Version History
+position: 5
+---
+
+The pages on this website always reflect the most recent and up-to-date version of the Beneficial Ownership Visualisation Standard (BOVS). You should always use the most recent version.
+{: .lead}
+
+A record of changes or additions to BOVS is shown below.
+
+# Version 0.3
```

Comment:
We should version separately from BODS. 

So this would be BOVS version 0.1-alpha or 0.1-beta to indicate, following https://semver.org/ that:

* This is subject to change at any time
* We're testing this, and will remove alpha or beta label once our testing is complete
* If we make changes following testing, we would increment version number.

Reply:
I don't think this is meant to align with BODS @timgdavies - Virgil simply chose version 0.3 based on the fact we'd already done two internal revisions. I'm happy to just go with 0.1, but I think the alpha/beta bit just complicates things - a version of 0.1 already makes it clear it's not a finished product.

Reply:
Will put back down to "0.1"